.github/workflows/build.yml

```diff
@@ -2,39 +2,109 @@ name: build
 
 on:
   push:
+    branches:
+      - main
   pull_request:
 
 jobs:
   build:
     runs-on: ubuntu-20.04
     steps:
-    - uses: actions/checkout@v3
-      name: checkout nosqlbench
-    - uses: actions/setup-java@v3
-      name: setup java
-      with:
-        java-version: '17'
-        java-package: jdk
-        architecture: x64
-        distribution: 'temurin'
-
-    - name: Cache Maven packages
-      uses: actions/cache@v1
-      with:
-        path: ~/.m2
-        key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
-        restore-keys: ${{ runner.os }}-m2
-
-    - name: mvn package
-      run: mvn package
-
-    - name: mvn verify
-      run: mvn verify
-
-    - name: Archive Test Results
-      if: always()
-      uses: actions/upload-artifact@v3
-      with:
-        name: test-results
-        path: |
-         [a-zA-Z]**/logs/*
+      - uses: actions/checkout@v3
+        name: checkout nosqlbench
+      - uses: actions/setup-java@v3
+        name: setup java
+        with:
+          java-version: '17'
+          java-package: jdk
+          architecture: x64
+          distribution: 'temurin'
+
+      - name: Cache Maven packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+
+      - name: mvn package
+        run: mvn package
+
+      - name: export docs
+        run: nb5/target/nb5 export-docs
+
+      - name: archive docs
+        uses: actions/upload-artifact@v3
+        with:
+          name: exported-docs
+          path: exported_docs.zip
+
+      - name: mvn verify
+        run: mvn verify
+
+      - name: Archive Test Results
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: test-results
+          path: |
+            [a-zA-Z]**/logs/*
+
+
+  docs:
+    needs: build
+    runs-on: ubuntu-20.04
+    steps:
+      - name: set git username
+        run: git config --global user.email "${{ secrets.NBDROID_EMAIL }}"
+
+      - name: set git email
+        run: git config --global user.name "${{ secrets.NBDROID_NAME }}"
+
+      - name: download exported-docs
+        uses: actions/download-artifact@v3
+        with:
+          name: exported-docs
+
+      - name: unzip docs
+        run: unzip exported_docs.zip
+
+      - run: ls -la
+
+      #      - name: download docs
+      #        uses: actions/download-artifact@v1
```

Comment:
We can go ahead and clean up old comments here and further down.

Reply:
Agreed!